titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/common/action/task/KillInitiatedActions.java

```diff
@@ -195,7 +196,8 @@ public static ChangeAction reconcilerInitiatedTaskKillInitiated(ReconciliationEn
                         jobStore,
                         V3JobOperations.Trigger.Reconciler,
                         reason,
-                        titusRuntime
+                        titusRuntime,
+                        CallMetadata.newBuilder().withCallerId("scheduler").build()
```

Comment:
For internal `CallMetadata` instances we should create constants, and populate all fields.